# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/media/webrtc/MediaTransportParent.h
# Commit: f5353ea9f1df
# Full Hash: f5353ea9f1dfe15a65076bc7aa272d688ebdb8ca
# Author: Byron Campen [:bwc] <docfaraday@gmail.com>
# Date: 2019-09-18 21:49:27
# Regressor Bug: 1569183
# File Overlap Count: 2
# Description:
#   Bug 1569183: Add a proxy policy argument to NrSocketProxyConfig, and always set this config on the NrIceCtx. r=mjf
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D45101
# ==============================================================================

diff -r ed1ae6a85255 -r f5353ea9f1df dom/media/webrtc/MediaTransportParent.h
--- a/dom/media/webrtc/MediaTransportParent.h	Tue Sep 17 18:17:13 2019 +0000
+++ b/dom/media/webrtc/MediaTransportParent.h	Tue Sep 17 18:17:14 2019 +0000
@@ -26,7 +26,8 @@
       const RTCIceTransportPolicy& icePolicy);
   mozilla::ipc::IPCResult RecvSetProxyConfig(const dom::TabId& tabId,
                                              const net::LoadInfoArgs& args,
-                                             const nsCString& alpn);
+                                             const nsCString& alpn,
+                                             const int& proxyPolicy);
   mozilla::ipc::IPCResult RecvEnsureProvisionalTransport(
       const string& transportId, const string& localUfrag,
       const string& localPwd, const int& componentCount);